app/export.go

```diff
@@ -62,6 +65,62 @@ var exportablePreferences = map[ComparablePreference]string{{
 }: "EmailInterval",
 }
 
+func (a *App) GetID() string {
+	return a.ID
+}
+func (a *App) SetID(id string) {
+	a.ID = id
+}
+
+func exportLogFile(attachment AttachmentImportData, exportFilename string) (LogFormat, *model.AppError) {
```

Comment:
Here can u change the return attribute from LogFormat to *LogFormat so that rather than returning empty struct we can return nil in error if condtions

Reply:
I can not return nil because on doing so I am getting an error that `cannot use nil as LogFormat value in return statementcompiler`